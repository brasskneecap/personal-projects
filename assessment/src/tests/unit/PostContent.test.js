import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import PostContent from '../../components/PostContent';

describe('PostContent', () => {
    const wrapper = new ShallowRenderer()

    wrapper.render(
    <PostContent content="This is a test" />
    );
    
    let componentRendered = wrapper.getRenderOutput();
    
    it('Interaction Snapshot', () => {
        expect(componentRendered).toMatchSnapshot();
    })

    it('Should contain the passed in content', () => {
        expect(componentRendered.props.children.props.children).toBe('This is a test');
    })
    
})