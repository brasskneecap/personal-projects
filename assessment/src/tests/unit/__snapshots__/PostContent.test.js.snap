// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostContent Interaction Snapshot 1`] = `
<div
  className="post-content"
>
  <p>
    This is a test
  </p>
</div>
`;
